internal/provider/resource_compute_virtual_machine.go

```diff
@@ -67,6 +67,31 @@ Virtual machines can be created using three different methods:
 					Type: schema.TypeString,
 				},
 			},
+			"cloud_init": {
+				Type:     schema.TypeMap,
+				ForceNew: true,
+				Optional: true,
+				Description: `A set of cloud-init compatible key/value used to configure the virtual machine.
+					
+	List of cloud-init compatible keys :
+	- ` + "`user-data` (This value value should be base64 encoded)" + `
+	- ` + "`network-config` (This value value should be base64 encoded)" + `
```

Comment:
the word `value` is duplicated

```suggestion
	- ` + "`user-data` (This value should be base64 encoded)" + `
	- ` + "`network-config` (This value should be base64 encoded)" + `
```